packages/sanity/src/core/form/inputs/ReferenceInput/ReferenceInput.tsx

```diff
@@ -252,8 +260,31 @@ export function ReferenceInput(props: ReferenceInputProps) {
       })),
     [searchState.hits]
   )
+
+  const isEditing = focusPath.length === 1 && (focusPath[0] === '_ref' || focusPath[0] === '$')
```

Comment:
I don't think we need to check for `$` anymore(?) since we got rid of that idiosyncrasy in v3